configure.ac

```diff
@@ -770,7 +771,7 @@ if test "X$BUILD_XDR" != "Xyes"; then
 
   AC_LINK_IFELSE([AC_LANG_PROGRAM([[
   #include <rpc/types.h>
-  #include <rpc/xdr.h>]], [[xdr_int]])],[AC_MSG_RESULT([yes]); BUILD_XDR="no"],[AC_MSG_RESULT([no]); BUILD_XDR="yes"])
+  #include <rpc/xdr.h>]], [[xdr_int]])],[BUILD_XDR="no"],[BUILD_XDR="yes"])
```

Comment:
This resulted in a hanging "yes" or "no" in the configure output since the check it's intended for finished several lines up.